photon-lib/src/main/native/cpp/photon/PhotonCamera.cpp

```diff
@@ -200,9 +200,33 @@ void PhotonCamera::VerifyVersion() {
           cameraNameOutString);
     }
   } else if (!VersionMatches(versionString)) {
-    FRC_ReportError(frc::warn::Warning,
+    std::string bfw = "";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>    ____  _________    ____     ________  ___________    __  \n";
+    bfw += ">>>   / __ \\/ ____/   |  / __ \\   /_  __/ / / /  _/ ___/   / /\n";
+    bfw += ">>>  / /_/ / __/ / /| | / / / /    / / / /_/ // / \\__ \\   / / \n";
+    bfw += ">>> / _, _/ /___/ ___ |/ /_/ /    / / / __  // / ___/ /  /_/    \n";
+    bfw += ">>>/_/ |_/_____/_/  |_/_____/    /_/ /_/ /_/___//____/  (_)     \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> You are running an incompatible version                     \n";
+    bfw += ">>> of PhotonVision on your coprocessor!                        \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> This is neither tested nor supported.                       \n";
+    bfw += ">>> You MUST update either PhotonVision, PhotonLib, or both.    \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> Your code will now crash. We hope your day gets better.     \n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
```

Comment:
The following is more efficient and has less visual noise. The same should work in Java.
```suggestion
    std::string_view bfw =
      ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
      ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
      ">>>    ____  _________    ____     ________  ___________    __  \n"
      ">>>   / __ \\/ ____/   |  / __ \\   /_  __/ / / /  _/ ___/   / /\n"
      ">>>  / /_/ / __/ / /| | / / / /    / / / /_/ // / \\__ \\   / / \n"
      ">>> / _, _/ /___/ ___ |/ /_/ /    / / / __  // / ___/ /  /_/    \n"
      ">>>/_/ |_/_____/_/  |_/_____/    /_/ /_/ /_/___//____/  (_)     \n"
      ">>>                                                             \n"
      ">>> You are running an incompatible version                     \n"
      ">>> of PhotonVision on your coprocessor!                        \n"
      ">>>                                                             \n"
      ">>> This is neither tested nor supported.                       \n"
      ">>> You MUST update either PhotonVision, PhotonLib, or both.    \n"
      ">>>                                                             \n"
      ">>> Your code will now crash. We hope your day gets better.     \n"
      ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
      ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";

```